TASK-6/ShopLogic.cs

```diff
@@ -0,0 +1,57 @@
+using Newtonsoft.Json;
+using NLog;
+
+namespace TASK_6;
+
+public class ShopLogic
+{
+    public static JsonShops JsonShops { get; set; }
+    public static List<Phone> FoundPhones { get; set; }
+    private static string _path = "appsettings.json";
```

Comment:
Dont mix logics. Work with json - seperate classes, which return objects of our models. Place these classes, for example, in Services folder.